sycl/plugins/unified_runtime/README.md

```diff
@@ -0,0 +1,22 @@
+# Unified Runtime
+
+This directory contains the implementation of the PI plugin for Unified Runtime,
+including the pi2ur translation layer, as well as the sources for the individual
+Unified Runtime adapters.
+
+## Making changes to Unified Runtime
+If you introduce changes to PI (e.g. new entry points, new enum values) you
+should introduce matching changes to the Unified Runtime spec.
+
+To do this, open a Pull Request adding the changes to the
+[Unified Runtime](https://github.com/oneapi-src/unified-runtime)
+repository, making sure to follow the
+[Contribution Guide](https://oneapi-src.github.io/unified-runtime/core/CONTRIB.html).
+
+When your changes to Unified Runtime are merged, you should:
+* Update the UR commit used by changing the `UNIFIED_RUNTIME_TAG` value in
```

Comment:
We do not have enough CI testing in UR repo so we can't update regularly yet. This is coming in the next month or so, and `sycl/plugins/unified_runtime/ur` is moving to UR repo at the same time too.

Reply:
We do have weekly [pre-releases](https://github.com/oneapi-src/unified-runtime/releases) with named tags for this purpose. Its maybe something that could be automated, either that or we determine where that responsibility resides and go from there.

There's been quite a lot of UR changes this week to address open PR's with PI changes here but the order they were merged to UR might not match up. We're working on getting things as up to date as possible to hopefully avoid this situation recurring.

Reply:
For now, since we have so many changes going so fast, I think it is ok to update with just commits, but I agree, that in some point we should start updating only with pre-releases or stable tags as @kbenzie .

I dont think though that we need to have that done automatically in the future. In some point, UR spec would be very stable, so it's not likely spec will change significantly for even several weeks (now it is different due to working towards first version).